lsp-mode.el

```diff
@@ -7908,18 +7908,19 @@ in place."
 ;; VSCode marketplace
 
 (defcustom lsp-vscode-ext-url
-  "https://marketplace.visualstudio.com/_apis/public/gallery/publishers/%s/vsextensions/%s/%s/vspackage"
+  "https://marketplace.visualstudio.com/_apis/public/gallery/publishers/%s/vsextensions/%s/%s/vspackage%s"
   "Vscode extension template url."
   :group 'lsp-mode
   :type 'string
   :package-version '(lsp-mode . "8.0.0"))
 
-(defun lsp-vscode-extension-url (publisher name &optional version)
+(defun lsp-vscode-extension-url (publisher name version &optional targetPlatform)
```

Comment:
can you keep the version also optional?

Reply:
without the version number I couldn't get a proper link from the marketplace. I suppose that was also the reason why they were hardcoded to `lsp-lua.el`. I don't think they were ever optional.

Reply:
I think you can pass `latest` or something to get the latest version.
See https://github.com/emacs-lsp/dap-mode/blob/master/dap-utils.el#L72